release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1268,6 +1268,19 @@ In addition to the OCI image, the image-based upgrade uses the `ostree` library
 
 For more information, see xref:../edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc#understanding-image-based-upgrade-for-sno[Understanding the image-based upgrade for {sno} clusters].
 
+[id="ocp-4-16-image-based-upgrade-enhancements_{context}"]
+==== Image-based upgrade enhancements
+
+With this release, the image-based upgrade introduces the following enhancements:
+
+* Simplifies the upgrade process for a large group of managed clusters by adding the `ImageBasedGroupUpgrade` API on the hub
+* Labels the managed clusters for action completion when using the `ImageBasedGroupUpgrade` API
+* Improves seed cluster validation before the seed image generation
+* Automatically cleans up the container storage disk if usage reaches a certain threshold on the managed clusters
+* Adds comprehensive event history in the new `status.history` field of the `ImageBasedUpgrade` CR
+
+For more information about the `ImageBasedGroupUpgrade` API, see xref:../edge_computing/image_based_upgrade/ztp-image-based-upgrade.adoc#ztp-image-based-upgrade-concept_ztp-gitops[Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub].
```

Comment:
The link title should match the module title: Performing an image-based upgrade for single-node OpenShift clusters using GitOps ZTP

Reply:
This will link [here](https://docs.openshift.com/container-platform/4.17/edge_computing/image_based_upgrade/ztp-image-based-upgrade.html#ztp-image-based-upgrade-concept_ztp-gitops) but the main content is not merged yet